e2e/cjs/index.js

```diff
@@ -0,0 +1,7 @@
+const ns = require('node-schedule');
+
+if (!ns.isEOL('v0.10.48')) {
+  throw new Error('Expected v10 to be EOL');
+}
+
+console.log('CommonJS e2e successful');
```

Comment:
```suggestion
const nodeSchedule = require('node-schedule');

if (!nodeSchedule.isEOL('v0.10.48')) {
  throw new Error('Expected v10 to be EOL');
}

console.log('CommonJS e2e successful');
```

I find a style like this to be more readable, what do you think?

Reply:
```suggestion
const { isEOL }= require('node-schedule');

if (!isEOL('v0.10.48')) {
  throw new Error('Expected v10 to be EOL');
}

console.log('CommonJS e2e successful');
```

What about this? It's more like the esm version.

Reply:
That's also OK. I'll go make a PR for us to change this. 😉